reserver/reserver_param.py

```diff
@@ -21,4 +21,9 @@
     "url": r'^(http|https)://[a-zA-Z0-9.-_]+\.[a-zA-Z]{2,}(/\S*)?$',
 }
 INVALID_PACKAGE_PARAMETER_NAME_ERROR = "Given parameter doesn't exist among the supported user allowed parameters."
-INVALID_PACKAGE_PARAMETER_VALUE_ERROR = "Invalid value for {parameter} that should be a valid {regex}"
\ No newline at end of file
+INVALID_PACKAGE_PARAMETER_VALUE_ERROR = "Invalid value for {parameter} that should be a valid {regex}"
+INVALID_CONFIG_FILE_NAME_ERROR = "Given file name for user-defined setup.py params is not a string."
```

Comment:
Are you sure about `setup.py`?

Reply:
I update the error message, changing `setup.py` with `package`.